StateMachine stateMachine
string test
string[] inputTypes
---
string uppaalModel # xml
